Example/Tests/TransformationTests/CLDTransformationTests/CLDTransformationExpressionsTests.swift

```diff
@@ -142,6 +142,72 @@ class CLDTransformationExpressionsTests: BaseTestCase {
         XCTAssertEqual(actualResult, expectedResult, "Calling get x should return its value")
     }
     
+    func test_setStartOffset_emptyInputParamaters_shouldNotStoreNewVariable() {
+        
+        // Given
+        let input       = String()
+        let expression  = CLDExpression(value: input)
+        
+        // When
+        sut.setStartOffset(expression)
+        
+        let actualResult = sut.startOffset!
+        
+        // Then
+        XCTAssertTrue(actualResult.isEmpty, "Empty expression should not be stored in params")
+    }
+    
+    func test_setStartOffset_inputExpression_shouldStoreNewValue() {
+        
+        // Given
+        let input       = "duration - 3"
+        let expression  = CLDExpression(value: input)
+        
+        let expectedResult = "du_sub_3"
+        
+        // When
+        sut.setStartOffset(expression)
+        
+        let actualResult = sut.startOffset!
+        
+        // Then
+        XCTAssertFalse(actualResult.isEmpty, "actualy result should not be empty")
+        XCTAssertEqual(actualResult, expectedResult, "acutalyResult and expectedResult should be eqaul")
+    }
+    
+    func test_setEndOffset_emptyInputParamaters_shouldNotStoreNewVariable() {
+        
+        // Given
+        let input       = String()
+        let expression  = CLDExpression(value: input)
+        
+        // When
+        sut.setEndOffset(expression)
+        
+        let actualResult = sut.endOffset!
+        
+        // Then
+        XCTAssertTrue(actualResult.isEmpty, "Empty expression should not be stored in params")
+    }
+    
+    func test_setEndOffset_inputExpression_shouldStoreNewValue() {
+        
+        // Given
+        let input       = "duration - 3"
+        let expression  = CLDExpression(value: input)
+        
+        let expectedResult = "du_sub_3"
+        
+        // When
+        sut.setEndOffset(expression)
+        
+        let actualResult = sut.endOffset!
+        
+        // Then
+        XCTAssertFalse(actualResult.isEmpty, "actualy result should not be empty")
```

Comment:
Please fix the English error here :

`actualy result should not be empty` sounds odd (and has a typo)
while in the line below a space is missing and has a typo.

`acutalyResult and expectedResult should be eqaul`

Reply:
Fixed